vispy/app/backends/_qt.py

```diff
@@ -183,7 +198,7 @@ def message_handler(*args):
     resizable=True,
     decorate=True,
     fullscreen=True,
-    context=True,
+    context=not QT5_NEW_API or os.getenv(PYQT5_SHARE_ENV, False),
```

Comment:
`getenv` always returns a `str` so you could have `export VISPY_PYQT5_SHARE_CONTEXT=false` and this conditional will eval to `context=True`, you need to do a `str` comp like:
```
os.getenv(PYQT5_SHARE_ENV, 'false').lower() == 'true'
```

Reply:
Yes, of course. This was meant to be more of a hack, but I agree the more flexible thing would be to actually interpret the truthiness of the value.